.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea
+*.iml
```

Comment:
What is this file?

Reply:
https://www.jetbrains.org/intellij/sdk/docs/basics/project_structure.html
> Module Library: the library classes are visible only in this module and the library information is recorded in the module’s .iml file.